torch/_inductor/codegen/cpp_wrapper_gpu.py

```diff
@@ -288,22 +304,21 @@ def generate_load_kernel_once(
         keys = (get_cpp_wrapper_cubin_path_name(), "mangled_name", "shared_mem")
         kernel_var_name = f"kernels.{kernel_name}" if V.graph.aot_mode else kernel_name
         self.writeline(f"if ({kernel_var_name} == nullptr) {{")
-        self.writeline(
-            DeferredGpuKernelLine(
-                kernel_name,
-                (
-                    """    """
-                    + kernel_var_name
-                    + """ = loadKernel("%s", "%s", %s, this->cubin_dir_);"""
-                    if V.graph.aot_mode
-                    else """    """
-                    + kernel_var_name
-                    + """ = loadKernel("%s", "%s", %s);"""
-                ),
-                keys,
-            )
+        deferred_gpu_kernel_line = DeferredGpuKernelLine(
+            kernel_name,
+            (
+                """    """
+                + kernel_var_name
+                + """ = loadKernel("%s", "%s", %s, this->cubin_dir_);"""
+                if V.graph.aot_mode
+                else """    """ + kernel_var_name + """ = loadKernel("%s", "%s", %s);"""
```

Comment:
I know you just moving code around, but please consider at least replacing `""" """` with `" "` (and may be use f-strings)